jstest/common/utils.py

```diff
@@ -244,34 +244,36 @@ def copy(src, dst):
     '''
     Copy src to dst.
     '''
-    if not exists(src):
-        return
+    try:
+        if os.path.isdir(src):
+            # Remove dst folder because copytree function
+            # fails when is already exists.
+            if exists(dst):
+                shutil.rmtree(dst)
 
-    if os.path.isdir(src):
-        # Remove dst folder because copytree function
-        # fails when is already exists.
-        if exists(dst):
-            shutil.rmtree(dst)
+            shutil.copytree(src, dst, symlinks=False, ignore=None)
 
-        shutil.copytree(src, dst, symlinks=False, ignore=None)
+        else:
+            # Create dst if it does not exist.
+            basedir = dirname(dst)
+            if not exists(basedir):
+                mkdir(basedir)
 
-    else:
-        # Create dst if it does not exist.
-        basedir = dirname(dst)
-        if not exists(basedir):
-            mkdir(basedir)
+            shutil.copy(src, dst)
 
-        shutil.copy(src, dst)
+    except (IOError, OSError) as err:
```

Comment:
You don't need to handle exceptions because the [__main__.py (line 262)](https://github.com/Samsung/js-remote-test/blob/master/jstest/__main__.py#L232) handles all the exceptions.